internal/gamepad/api_windows_386.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !ebitencbackend && windows && 386
+// +build !ebitencbackend,windows,386
```

Comment:
and `arm` (32bit)?

Reply:
Resolved by your other suggestion (using a runtime check for both names in the DLL).